src/rules/selector-max-specificity/__tests__/index.js

```diff
@@ -177,3 +181,25 @@ testRule(rule, {
     description: "ignore rules with variable interpolation",
   }],
 })
+
+test("Cannot Parse Selector Warning", t => {
+  const config = {
+    rules: {
+      "selector-max-specificity": ["0,2,0"],
+    },
+  }
+  let planned = 0
+
+  postcss().use(stylelint(config)).process(".a, :matches(.b, .c, .d) e:first-child, .f > :patches(:first-child, :last-child) { }").then(result => {
+    t.equal(result.warnings().length, 1)
+    t.equal(result.warnings()[0].text, "cannot parse selector")
```

Comment:
Are line and column available here? If so, can we tests they they are correctly set please.